pg_search/src/bootstrap/test_table.rs

```diff
@@ -56,17 +73,19 @@ type MockDeliveryRow = (
 );
 
 #[pg_extern(sql = "
-CREATE OR REPLACE PROCEDURE paradedb.create_bm25_test_table(table_name VARCHAR DEFAULT 'bm25_test_table', schema_name VARCHAR DEFAULT 'paradedb', table_type paradedb.TestTable DEFAULT 'Items')
+CREATE OR REPLACE PROCEDURE paradedb.create_bm25_test_table(table_name VARCHAR DEFAULT 'bm25_test_table', schema_name VARCHAR DEFAULT 'paradedb', table_type paradedb.TestTable DEFAULT 'Items', create_mode paradedb.TestTableCreateMode DEFAULT 'TableAndRows')
 LANGUAGE c AS 'MODULE_PATHNAME', '@FUNCTION_NAME@';
 ", requires = [ TestTable ])]
```

Comment:
Based on a discussion with @eeeebbbbrrrr, I will likely revert this change and instead adjust the `CREATE INDEX` statement to use unusual segment limits to force multiple segments.

Reply:
I was not able to figure out how to craft a `layer_sizes` argument which would result in multiple segments during `CREATE INDEX`.

Let me know what you think of this API change for `create_bm25_test_table`... I think that it could be useful independent of this case.